<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="a">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="b">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="oblicz">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="wynik">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>70</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="operacje">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentText">
     <string>+</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>+</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>*</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>/</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="autor">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>70</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Pokaz autora</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="WypiszAutora">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>39</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="Kalendarz">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>140</y>
      <width>312</width>
      <height>183</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="WypiszKalendarz">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>340</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QScrollBar" name="horizontalScrollBar">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>400</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>430</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Podaj Nazwisko</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Nazwisko">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Dodaj">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Dodaj</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Notes">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>121</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
background-color: rgb(255, 255, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Notes</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="liczba">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>140</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Podaj liczbe</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Dodajv2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>160</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>121</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Dane</string>
    </property>
   </widget>
   <widget class="QLabel" name="pozycje">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>360</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="suma">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>380</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>400</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>wyczysc</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>360</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>liczba pozycji:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="dane2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>140</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>120</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Podaj liczbe</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>160</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>++</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zapis">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>zapis</string>
    </property>
   </widget>
   <widget class="QPushButton" name="odczyt">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>400</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>odczyt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pusty">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>430</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>pusty</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>360</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>kolor formularza</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="ButtonCzerwony">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>390</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>czerwony</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="ButtonZielony">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>410</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>zielony</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="ButtonSzary">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>430</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>szary</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menukalkulator">
    <property name="title">
     <string>Kalkulator</string>
    </property>
    <addaction name="actionPokaz"/>
    <addaction name="separator"/>
    <addaction name="actionUkryj"/>
   </widget>
   <widget class="QMenu" name="menuEdycja_notesu">
    <property name="title">
     <string>Edycja notesu</string>
    </property>
    <addaction name="actionKopiuj"/>
    <addaction name="separator"/>
    <addaction name="actionWytnij"/>
    <addaction name="separator"/>
    <addaction name="actionWklej"/>
   </widget>
   <widget class="QMenu" name="menuKomunikat">
    <property name="title">
     <string>Komunikat</string>
    </property>
    <addaction name="actionk1"/>
    <addaction name="separator"/>
    <addaction name="actionk2"/>
    <addaction name="separator"/>
    <addaction name="actionk3"/>
    <addaction name="separator"/>
    <addaction name="actionk4"/>
   </widget>
   <addaction name="menukalkulator"/>
   <addaction name="menuEdycja_notesu"/>
   <addaction name="menuKomunikat"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionPokaz">
   <property name="text">
    <string>Pokaz</string>
   </property>
  </action>
  <action name="actionUkryj">
   <property name="text">
    <string>Ukryj</string>
   </property>
  </action>
  <action name="actionKopiuj">
   <property name="text">
    <string>Kopiuj</string>
   </property>
  </action>
  <action name="actionWytnij">
   <property name="text">
    <string>Wytnij</string>
   </property>
  </action>
  <action name="actionWklej">
   <property name="text">
    <string>Wklej</string>
   </property>
  </action>
  <action name="actionk1">
   <property name="text">
    <string>k1</string>
   </property>
  </action>
  <action name="actionk2">
   <property name="text">
    <string>k2</string>
   </property>
  </action>
  <action name="actionk3">
   <property name="text">
    <string>k3</string>
   </property>
  </action>
  <action name="actionk4">
   <property name="text">
    <string>k4</string>
   </property>
  </action>
  <action name="actionk5">
   <property name="text">
    <string>k5</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
